index.js

```diff
@@ -59,28 +59,28 @@ function makeVarMap(filename) {
     // extracting values (depth-first, post-order traversal), files that import
     // libraries can re-define variable declarations, which more-closely
     // matches the browser's behavior
-    style.root.eachAtRule(function(atRule) {
+    style.root.eachAtRule(function (atRule) {
       if (atRule.name !== 'import')
         return;
 
       var stripped = atRule.params.replace(/^["']/, '').replace(/['"]$/, '');
 
-      process(resolveImport(stripped, dirname(filename)));
+      process(resolveImport(stripped, filename));
```

Comment:
this seems wrong -- the second argument for `resolveImport` needs to name a directory, not just any file.